### DISCUSSION TITLE
Error when manually deleting events

### DISCUSSION THREAD

**Discussion by Tthongsy** [datalust/seq-tickets#1367] [Help]
![SeqDeleteEvent](https://user-images.githubusercontent.com/88148249/139695997-dbb96be0-4e5a-4179-9f78-2184c0b0ea64.PNG)

Hi I'm getting this error when manually deleting events and I'm unsure what to do because the events themselves are still visible within the Seq web portal.

Thanks,
Tommy T

**Comment by nblumhardt** [comment]
Hi Tommy, thanks for the heads-up!

There should be some further diagnostic info in Settings > Diagnostics > Download a full diagnostic report. If you're able to spot any clues in there, or copy it to `support@datalust.co` that should narrow it down.

It would be great to confirm also which version this instance is running.

Best regards,
Nick

**Reply by Tthongsy** [reply]
Sure, should I send in a full diagnostic in txt format with anything specific in the subject line?

**Reply by nblumhardt** [reply]
Received and replied now - thanks!
